src/test/java/cz/czechitas/AdminPage.java

```diff
@@ -0,0 +1,29 @@
+package cz.czechitas;
+
+import org.openqa.selenium.*;
+
+public class AdminPage {
+
+    WebDriver driver;
+
+    public AdminPage(WebDriver driver) {
+        this.driver = driver;
+
+    }
+
+    public void openPage() throws InterruptedException {
+        driver.navigate().to("http://czechitas-shopizer.azurewebsites.net/admin/logon.html");
+        Thread.sleep(5000L);
```

Comment:
Toto neni dobry napad. Kazdy beh testu to zpomali o 5 sekund, predstav si ze login volas v 200 testech ... to je 16 minut navic uplne zbytecne jen na otvirani stranky.

Reply:
Mně to bez toho nešlo, mám pomalý pc.

Reply:
V tom pripade je lepsi implementovat cekani na nejaky element, napriklad muzes implementovat cekani na ten username field v **AdminPage** v te login **metode**, pomoci toho **WebDriverWait** https://www.selenium.dev/documentation/en/webdriver/waits/

Je lepsi cekat az kdyz skutecne potrebujes se strankou interagovat, jinak se ti ty testy strasne prodlouzi.,